server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -132,5 +143,64 @@ boolean checkIsHealthy() {
       }
       return isHealthy;
     }
+
+    boolean checkStorageReadAndWrite() {
+      if (storage.isCorrupted()) {
+        return false;
+      }
+      File checkDir = new File(storageDir, "check");
+      try {
+        checkDir.mkdirs();
+        File writeFile = new File(checkDir, "test");
+        if (!writeFile.createNewFile()) {
+          return false;
+        }
+        try (FileOutputStream fos = new FileOutputStream(writeFile)) {
+          for (int i = 0; i < 5; i++) {
+            fos.write(0xa);
+            fos.write(0x5);
+          }
+        }
+        char[] readData = new char[10];
```

Comment:
It'd be better to generate and a list of random bytes, say 10 random bytes, and verify them.

Reply:
OK